docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md

```diff
@@ -381,4 +389,10 @@ docker run --rm --env-file /path/to/env/file/configs.env -it snowplow/fractribut
 </TabItem>
 </Tabs>
 
+:::tip
+
+To help debug easier we have included logs to be printed each time a table is created in the warehouse, there should be 3 in total (`snowplow_fractribution_path_summary_with_channels`, `snowplow_fractribution_channel_attribution` and `snowplow_fractribution_report_table`). There is also a `-- verbose` flag you can include in your shell prompt when executing the python scripts which may give you an even more detailed breakdown depending on your warehouse.
+
+:::
```

Comment:
Is this how you pass it in for the docker approach, or does it need to be an env variable?